cocotb/binary.py

```diff
@@ -281,12 +281,30 @@ def _adjust_twos_comp(self, x):
             rv = x
         return rv
 
-    def get_value(self):
-        """Return the integer representation of the underlying vector."""
+    @property
+    def value(self):
+        """Integer access to the value. **deprecated**"""
+        return self.integer
+
+    @value.setter
+    def value(self, val):
+        self.integer = val
+
+    @property
+    def integer(self):
+        """The integer representation of the underlying vector."""
         return self._convert_from[self.binaryRepresentation](self._str)
 
-    def get_value_signed(self):
-        """Return the signed integer representation of the underlying vector."""
+    @integer.setter
+    def integer(self, val):
+        self._str = self._convert_to[self.binaryRepresentation](val)
+
+    get_value = value.fget
+    set_value = value.fset
```

Comment:
The nittiest of nitpicks, this should probably be moved up to directly below the `value` property definitions.

Reply:
Better to do the reverse and move the property down: https://github.com/cocotb/cocotb/pull/1497#discussion_r395069321